admin/src/templates/edit_user.html

```diff
@@ -36,11 +36,23 @@
       <span class="help-block">{{errors.phone}}</span>
     </div>
 
-    <div class="form-group required" mm-auth="can_configure" ng-class="{'has-error': errors.role}">
-      <label for="role" translate>configuration.role</label>
-      <input type="text" class="form-control" ng-model="editUserModel.role" disabled="disabled" ng-if="editUserModel.role === '_admin'" />
-      <select id="role" class="form-control" ng-options="key as value.name | translate for (key, value) in roles" ng-model="editUserModel.role" ng-if="editUserModel.role !== '_admin'"></select>
-      <span class="help-block">{{errors.role}}</span>
+    <div id="role-select" class="form-group" mm-auth="can_configure" ng-class="{'has-error': errors.role}">
```

Comment:
Sorry I didn't see this earlier. 
```suggestion
    <div id="role-select" class="form-group" mm-auth="can_configure" ng-class="{'has-error': errors.roles}">
```

Without this, the error appears in grey: 
![image](https://user-images.githubusercontent.com/35681649/202651718-689cccda-84b1-4d4c-b231-414e87c1609b.png)

Reply:
Fixed.